pkg/worker/redis.go

```diff
@@ -136,7 +136,7 @@ func (w *RedisWorker) dequeueLoop(ctx context.Context, i, total int) {
 
 	// spread polling intervals
 	delayMs := (int(w.pollInterval.Milliseconds()) / total) * (i - 1)
-	logger.Info().Msgf("Worker start delay %dms", delayMs)
+	logger.Debug().Msgf("Worker start delay %dms", delayMs)
```

Comment:
This is a similar approach so I am syncing the logger level to debug for both messages.